From dfd70347efff7e27a368e7c887a4e563bdccb106 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 14 Sep 2023 08:50:21 +0200
Subject: [PATCH 2/2] com.palm.configurator.role.json.in: Add fixes for various
 permission issues

Solves:

2022-12-09T22:31:11.633364Z [4.946651635] kern.warning ls-hubd [] ls-hubd LSHUB_ROLE_FILE {} Can not find service "com.palm.imlibpurple" permissions for executable "/usr/sbin/configurator"
2022-12-09T22:31:11.731384Z [5.044671657] kern.warning ls-hubd [] ls-hubd LSHUB_ROLE_FILE {} Can not find service "com.palm.service.contacts" permissions for executable "/usr/sbin/configurator"
2022-12-09T22:31:11.732398Z [5.045685809] kern.warning ls-hubd [] ls-hubd LSHUB_ROLE_FILE {} Can not find service "org.webosports.service.messaging" permissions for executable "/usr/sbin/configurator"
2022-12-09T22:31:11.733725Z [5.047012721] kern.warning ls-hubd [] ls-hubd LSHUB_ROLE_FILE {} Can not find service "com.palm.service.contacts.linker" permissions for executable "/usr/sbin/configurator"
2022-12-09T22:31:11.736186Z [5.049472690] kern.warning ls-hubd [] ls-hubd LSHUB_ROLE_FILE {} Can not find service "*" permissions for executable "/usr/sbin/configurator"
2022-12-09T22:31:11.738608Z [5.051896281] kern.warning ls-hubd [] ls-hubd LSHUB_ROLE_FILE {} Can not find service "*" permissions for executable "/usr/sbin/configurator"
2022-12-09T22:31:11.774231Z [5.087518171] kern.warning ls-hubd [] ls-hubd LSHUB_ROLE_FILE {} Can not find service "com.palm.service.contacts.linker" permissions for executable "/usr/sbin/configurator"
2022-12-09T22:31:12.949996Z [6.263283706] kern.warning ls-hubd [] ls-hubd LSHUB_ROLE_FILE {} Can not find service "com.palm.service.contacts.linker" permissions for executable "/usr/sbin/configurator"

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
Upstream-Status: Pending
---
 files/sysbus/com.webos.service.configurator.perm.json | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.configurator.perm.json b/files/sysbus/com.webos.service.configurator.perm.json
index bb71bdf..27289ca 100644
--- a/files/sysbus/com.webos.service.configurator.perm.json
+++ b/files/sysbus/com.webos.service.configurator.perm.json
@@ -20,6 +20,14 @@
         "networking.query",
         "applications.query",
         "tts.internal",
-        "ppm.private"
+        "ppm.private",
+        "application.launcher",
+        "calendarremindersservice.operation",
+        "contactslinkerservice.operation",
+        "contactsservice.operation",
+        "imlibpurple-service.operation",
+        "messaging-service.operation",
+        "telephony.management",
+        "update-service.operation"
     ]
 }
-- 
2.41.0.windows.3

